gpioioctl/basic_test.go

```diff
@@ -0,0 +1,120 @@
+package gpioioctl
```

Comment:
The usual Go style is:

```
// Copyright 2024 The Periph Authors. All rights reserved.
// Use of this source code is governed under the Apache License, Version 2.0
// that can be found in the LICENSE file.

// Basic tests. More complete test is contained in the
// periph.io/x/cmd/periph-smoketest/gpiosmoketest
// folder.

package gpioioctl

import (
```

Note that cmd is not versioned, since it's a set of executables, not an API.